integration-tests/src/lib.rs

```diff
@@ -1,3 +1,16 @@
+#![cfg_attr(dylint_lib = "amplifier_lints", allow(cosmwasm_addr_in_msg_struct))]
+#![cfg_attr(
+    dylint_lib = "amplifier_lints",
+    allow(execute_without_explicit_permissions)
+)]
+#![cfg_attr(dylint_lib = "amplifier_lints", allow(get_fn))]
+#![cfg_attr(
+    dylint_lib = "amplifier_lints",
+    allow(msg_without_explicit_permissions)
+)]
+#![cfg_attr(dylint_lib = "amplifier_lints", allow(ref_opt_type))]
+#![cfg_attr(dylint_lib = "amplifier_lints", allow(unwraps_outside_tests))]
```

Comment:
Would this work?
```suggestion
#![cfg_attr(dylint_lib = "amplifier_lints", allow(cosmwasm_addr_in_msg_struct, execute_without_explicit_permissions, get_fn, msg_without_explicit_permissions, ref_opt_type, unwraps_outside_tests))]
```
Are all of these even necessary? We want to allow unwrap because it makes tests more concise, but I don't think that we need to allow any of the others